static/js/message_events.js

```diff
@@ -136,16 +137,23 @@ exports.update_messages = function update_messages(events) {
         }
 
         const new_topic = util.get_edit_event_topic(event);
+        // TODO: notify and move messages whenever message's topics are moved
+        const {new_stream_id} =  event;
```

Comment:
Fix the whitespace here.  Also is the TODO still incomplete?

Reply:
Nope, the TODO is fixed. I forgot to remove it